desktop.blocks/page/__conditional-comment/page__conditional-comment.bemhtml

```diff
@@ -0,0 +1,23 @@
+block('page').elem('conditional-comment')(
+    tag()(false),
+
+    content()(function() {
+        var ctx = this.ctx,
+            cond = ctx.condition
+                .replace('<', 'lt')
+                .replace('>', 'gt')
+                .replace('=', 'e'),
+            hasNegation = cond.indexOf('!') > -1,
+            includeOthers = ctx.includeOthers,
```

Comment:
I think that default value should be `false`, so the behaviour shouldn't change. I mean 
```js
includeOthers = ctx.msieOnly === true
```